karate-core/src/main/java/com/intuit/karate/XmlUtils.java

```diff
@@ -129,13 +129,7 @@ public static Document toHtmlDoc(String html) {
             builder.setEntityResolver(dtdEntityResolver);
             org.jsoup.nodes.Document jsoupDoc = Jsoup.parse(html);
             Document doc = new W3CDom().fromJsoup(jsoupDoc);
```

Comment:
comment was incorrect -- html does have dtds but not sure that we need to strip doctype here